File: retos-ts/06_regalos_caja.ts
// Resultado 5 ⭐
function inBox(box: string[]): boolean {
    for (let i = 1; i < box.length - 1; i++) {
        const line: string = box[i]
        const j: number = line.indexOf('*');    
        
        if(j !== 0 && j !== line.length - 1 && j !== -1 ){
            return true
        }
    }

    return false
}

File: retos-python/06_regalos_caja.py
# Resultado 5 ⭐
def in_box(box):
    for i in range(1, len(box) - 1):
        line = box[i]
        j = line.find('*')
        
        if j != 0 and j != len(line) - 1 and j != -1:
            return True

    return False